<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GNUFreeCall</class>
 <widget class="QMainWindow" name="GNUFreeCall">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>480</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>GNUFreeCall</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="pageMain">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QGraphicsView" name="videoView">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="sizeIncrement">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="verticalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOff</enum>
            </property>
            <property name="horizontalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOff</enum>
            </property>
            <property name="sceneRect">
             <rectf>
              <x>0.000000000000000</x>
              <y>0.000000000000000</y>
              <width>0.000000000000000</width>
              <height>0.000000000000000</height>
             </rectf>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="renderHints">
             <set>QPainter::Antialiasing</set>
            </property>
            <property name="transformationAnchor">
             <enum>QGraphicsView::NoAnchor</enum>
            </property>
            <property name="resizeAnchor">
             <enum>QGraphicsView::NoAnchor</enum>
            </property>
            <property name="viewportUpdateMode">
             <enum>QGraphicsView::NoViewportUpdate</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonYou">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>You</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>icons/you_icon.png</normaloff>icons/you_icon.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>72</width>
              <height>72</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonFellowGNUs">
            <property name="text">
             <string>Fellow GNUs</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>icons/fellow_gnus_icon.png</normaloff>icons/fellow_gnus_icon.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>72</width>
              <height>72</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonDiscoverHurd">
            <property name="text">
             <string>Discover Hurd</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>icons/discover_hurd_icon.png</normaloff>icons/discover_hurd_icon.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>72</width>
              <height>72</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageYou">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="font">
               <font>
                <pointsize>16</pointsize>
               </font>
              </property>
              <property name="text">
               <string>You</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="baseSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap>icons/you_icon.png</pixmap>
              </property>
              <property name="scaledContents">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLineEdit" name="searchYou">
            <property name="inputMask">
             <string/>
            </property>
            <property name="text">
             <string>Search...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTreeWidget" name="treeYou">
            <column>
             <property name="text">
              <string>Cloud Name</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>You</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>GNUs</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Hurd</string>
             </property>
            </column>
            <item>
             <property name="text">
              <string>Name</string>
             </property>
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Address</string>
             </property>
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Interests</string>
             </property>
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Pictures</string>
             </property>
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QPushButton" name="buttonAddCloud">
              <property name="text">
               <string>+ Add Cloud</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageFellowGNUs">
       <widget class="QWidget" name="verticalLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>424</width>
          <height>742</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Fellow GNUs</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="baseSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap>icons/fellow_gnus_icon.png</pixmap>
             </property>
             <property name="scaledContents">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLineEdit" name="searchFellowGNUs">
           <property name="inputMask">
            <string/>
           </property>
           <property name="text">
            <string>Search...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTreeWidget" name="treeFellowGNUs">
           <column>
            <property name="text">
             <string>Cloud Name</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>You</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>GNUs</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Hurd</string>
            </property>
           </column>
           <item>
            <property name="text">
             <string>David Sugar</string>
            </property>
            <property name="font">
             <font>
              <pointsize>16</pointsize>
             </font>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Hospital</string>
            </property>
            <property name="font">
             <font>
              <pointsize>16</pointsize>
             </font>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Andrew Bovsunovskiy</string>
            </property>
            <property name="font">
             <font>
              <pointsize>16</pointsize>
             </font>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Pimm Hogeling</string>
            </property>
            <property name="font">
             <font>
              <pointsize>16</pointsize>
             </font>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QPushButton" name="buttonAddGNU">
             <property name="text">
              <string>+ Add GNU</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="pageDiscoverHurd">
       <widget class="QWidget" name="verticalLayoutWidget_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>424</width>
          <height>742</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Discover Hurd</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="baseSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap>icons/discover_hurd_icon.png</pixmap>
             </property>
             <property name="scaledContents">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLineEdit" name="searchDiscoverHurd">
           <property name="inputMask">
            <string/>
           </property>
           <property name="text">
            <string>Search...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTreeWidget" name="treeDiscoverHurd">
           <column>
            <property name="text">
             <string>Cloud Name</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>You</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>GNUs</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Hurd</string>
            </property>
           </column>
           <item>
            <property name="text">
             <string>David Sugar</string>
            </property>
            <property name="font">
             <font>
              <pointsize>16</pointsize>
             </font>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Andrew Bovsunovskiy</string>
            </property>
            <property name="font">
             <font>
              <pointsize>16</pointsize>
             </font>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Pimm Hogeling</string>
            </property>
            <property name="font">
             <font>
              <pointsize>16</pointsize>
             </font>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Hospital A</string>
            </property>
            <property name="font">
             <font>
              <pointsize>16</pointsize>
             </font>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Hospital B</string>
            </property>
            <property name="font">
             <font>
              <pointsize>16</pointsize>
             </font>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Astronaut №1</string>
            </property>
            <property name="font">
             <font>
              <pointsize>16</pointsize>
             </font>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QPushButton" name="buttonListView">
             <property name="text">
              <string>List View</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_6">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="buttonCloudView">
             <property name="text">
              <string>Cloud View</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
